.gitignore

```diff
@@ -76,6 +76,10 @@ node_modules/
 # Flask Session files
 /origin-bridge/*flask_session/
 
+# origin-discovery #
+####################
+origin-discovery/continueTest
```

Comment:
I think we should have the uni test delete this file (I can do that in a separate PR. ok to leave this in for now...)

Reply:
Its a quick and minor change.... a solution in this PR: https://github.com/OriginProtocol/origin/pull/902